react/src/components/search-app/SearchBarComponent.jsx

```diff
@@ -28,12 +28,12 @@ export default class SearchBarComponent extends React.Component {
     this.setState({ query });
     if (query.length >= 2) {
       const http = new XMLHttpRequest();
-      const params = JSON.stringify({ query });
-      http.open('POST', '/api/v1/search', true);
+      http.open('GET', `${this.props.host}/search?query=${encodeURIComponent(query)}&mode=auto-suggest`, true);
```

Comment:
My idea was to build it as close to the architecture in #21 as possible. That means, that the components can be included in every site and are decoupled from the server (it should be theoretically possible that the API-server and data can be cloned by others and hosted on other IPs). Thus, the client-site-developer should get the possibility to change the host easily.

Later, I would prefer a syntax like:
```
<div id="abc-component" />
<script>
music.AbcComponent('#abc-component', {
    host: '...',
    ...
});
</script>
```

And, I think that passing the host via `<div ... data-host="..." />` is closer to it than writing a constants module.